src/server/http/index.js

```diff
@@ -41,6 +41,7 @@ export default async function (kbnServer, server, config) {
   server.connection({
     host: config.get('server.host'),
     port: config.get('server.port'),
+    tls: config.get('server.ssl.enabled'),
```

Comment:
The comment above says that these values are the same as the new platform configuration, but that's not the case with this tls config. Here `tls` is a boolean, but in the new platform it's an object with properties. Can you either make them the same or update the comment so this doesn't look like a bug in the future?

Reply:
Yep, sure, I'll update comment